ner_v2/api.py

```diff
@@ -557,7 +559,143 @@ def phone_number(request):
+                            "fallback_value": null,
+                            "predetected_values": ["Mumbai"],
+                            "fuzziness": null,
+                            "min_token_len_fuzziness": null,
+                            "use_fallback": false
+                        },
+                        "restaurant": {
+                            "structured_value": null,
+                            "fallback_value": null,
+                            "predetected_values": null,
+                            "fuzziness": null,
+                            "min_token_len_fuzziness": null,
+                            "use_fallback": false
+                                }
+                             }
+                         }
+                output response:
+                    {
+                        "success": true,
+                        "error": null,
+                        "data": [
+                            {
+                            "entities": {
+                                "restaurant": [],
+                                "city": [
+                                    {
+                                        "entity_value": {
+                                            "value": "New Delhi",
+                                            "datastore_verified": true,
+                                            "model_verified": false
+                                        },
+                                        "detection": "structure_value_verified",
+                                        "original_text": "delhi",
+                                        "language": "en"
+                                    },
+                                    {
+                                        "entity_value": {
+                                            "value": "Mumbai",
+                                            "datastore_verified": false,
+                                            "model_verified": true
+                                        },
+                                        "detection": "structure_value_verified",
+                                        "original_text": "Mumbai",
+                                        "language": "en"
+                                    }
+                                ]
+                                },
+                                "language": "en"
+                            }
+                        ]
+                    }
     """
-    return HttpResponse(status=500)
+    data = []
+
+    if request.method == "GET":
+        response = {"success": False, "error": "Get method is not allowed"}
+        return HttpResponse(json.dumps(response), status=501)
+
+    elif request.method == "POST":
+        ner_logger.debug("Fetching result")
+
+        try:
+            verify_text_request(request)
+            # if verify success parse request and get data
+            data = parse_text_request(request)
```

Comment:
Should be seperate out parsing and get data logic ? 
According to function definition it should do parsing job.

Reply:
Agree, `verify_text_request` is parsing and verifying the contents.
Changed the name of function and it's definition.